### PR TITLE
#21: Fixed unescaped password

### LINKED ISSUE #21: Creation fails if password contains special characters
For certain passwords the resource creation fails:

```
│ Error: local-exec provisioner error
│ 
│   with module.exasol.null_resource.exasol_cluster_wait[0],
│   on ../../terraform-aws-exasol/main.tf line 72, in resource "null_resource" "exasol_cluster_wait":
│   72:   provisioner "local-exec" {
│ 
│ Error running command '    python3 ../../terraform-aws-exasol/scripts/exasol_xmlrpc.py \
│       --license-server-address \
│       18.196.183.63 \
│       --username admin \
│       --password Z+Dia7D>w]NDS)hT
│ ': exit status 2. Output: /bin/sh: 5: Syntax error: ")" unexpected
```
_No worries / hope, even so the password is a real example it's no longer valid ;-)_

### LINKED ISSUE #23: Add integration tests
Add integration tests

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #21
Closes #23

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## Usage:
+
+* Install Terraform and Go
+* Get a exasol license and store it as `simple_exasol_setup/exasol.lic`
+* Create the file `simple_exasol_setup/terraform.tfvars` and fill in:
+  ```
+  owner = "<YOUR_EMAIL>"
+  aws_profile = "<AWS PROFILE TO USE>"
+  ```
+* run `go test --timeout 2h`
```
Comment (morazow) [inline test/README.md:14]:
```suggestion
* Run `go test --timeout 2h`
```

File: test/go.mod [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module gihub.exasol.com/terraform-aws-exasol/test
+
+  go 1.16
+
+  require (
+  github.com/grantstreetgroup/go-exasol-client v0.0.0-20210526002632-296e16acd559
+  github.com/gruntwork-io/terratest v0.35.4
+  github.com/sethvargo/go-password v0.2.0
+  github.com/stretchr/testify v1.7.0
+  )
```
Comment (morazow) [inline test/go.mod:10]: Is indentation off in this file?

File: test/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Tests for terraform-aws-exasol
+
+This directory contains tests for the Exasol terraform module
```
Comment (morazow) [inline test/README.md:3]:
```suggestion
This directory contains tests for the Exasol terraform module.
```

File: main.tf [hunk 4]
```diff
@@ -72,11 +72,14 @@ resource "null_resource" "exasol_cluster_wait" {
...
-      ${data.aws_instance.management_server.public_ip} \
+      --license-server-address "$IP"\
       --username admin \
-      --password ${var.admin_user_password}
+      --password "$ADMIN_PASS"
   EOF
+    environment = {
+      IP         = data.aws_instance.management_server.public_ip
+      ADMIN_PASS = var.admin_user_password
+    }
```
Comment (morazow) [inline main.tf:82]: :+1:

**Review by morazow — APPROVED** [review]
Approved with minor suggestions.

File: test/go.mod (hunk 2 above)
Comment (jakobbraun) [reply test/go.mod:10]: I think go formats all files. So probably it's intended to be like that.

**Pull request merged**